src/libasr/codegen/asr_to_c_cpp.h

```diff
@@ -2775,6 +2775,7 @@ PyMODINIT_FUNC PyInit_lpython_module_)" + fn_name + R"((void) {
             SET_INTRINSIC_NAME(Exp, "exp");
             SET_INTRINSIC_NAME(Exp2, "exp2");
             SET_INTRINSIC_NAME(Expm1, "expm1");
+	    SET_INTRINSIC_NAME(Trunc, "trunc");
```

Comment:
Please fix all the indents to use spaces.
We use 4 spaces as a tab.

Reply:
Sure, done. Thanks :-)